src/redux/reducer.js

```diff
@@ -16,13 +16,13 @@ const getGreetingsSucces = () => ({
 
 const getGreetingsError = () => ({
   type: GET_GREETING_ERROR,
-  payload: "Error",
+  payload: 'Error',
 });
 
 // Thunk
 
 const getGreeting = () => async (dispatch) => {
-  const response = await axios.get("http://localhost:3001/greetings");
+  const response = await axios.get('http://localhost:3001/api/greetings');
   const greeting = response.data;
```

Comment:
- [ ] [OPTIONAL] Greate work creating the app however you should get only the message from the API so your app is working 
```suggestion
  const greeting = response.data.message;
```